jpo-ode-svcs/src/test/resources/us/dot/its/jpo/ode/services/asn1/expected-bsm.json

```diff
@@ -71,13 +68,123 @@
           "brakeBoost": "unavailable",
           "auxBrakes": "unavailable"
         },
-        "size": {
-          "width": 190,
-          "length": 570
-        }
+        "size": { "width": 190, "length": 570 }
       },
-      "partII": []
+      "partII": [
```

Comment:
@drewjj here is where that `partII` data is now expected with some of the serialization updates.

Reply:
Nice, this looks great